browser_history/cli.py

```diff
@@ -3,11 +3,12 @@
 
 import sys
 import argparse
-from browser_history import get_history, generic, browsers
+from browser_history import get_history, generic, browsers,get_bookmarks
```

Comment:
Would it be better if `get_history` and `get_bookmarks` are placed next to each other? Just a suggestion :smiley:

Reply:
Yea sure